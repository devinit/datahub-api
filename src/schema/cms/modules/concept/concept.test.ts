import 'jest';
import {getConcepts, IConcept, getConceptAsync, getConcept} from '.';

describe('Github concept data tests', () => {
    it('should get data from concept.csv of a module on github', async () => {
        const conceptsData: IConcept[] = await getConcepts('global-picture');
        const concept: IConcept = getConcept('avg_income_of_extreme_poor', conceptsData);
        expect(conceptsData.length).toBeGreaterThan(2);
        expect(conceptsData[0].id).toBe('avg_income_of_extreme_poor');
    }, );
    it('should get data from concept.csv of an Id in a module', async () => {
        const concept: IConcept = await getConceptAsync('global-picture', 'avg_income_of_extreme_poor');
        expect(concept.id).toBe('avg_income_of_extreme_poor');
    }, );
});
