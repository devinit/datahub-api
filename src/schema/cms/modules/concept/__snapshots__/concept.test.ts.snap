// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Github concept data tests should get methodology data 1`] = `"Array []"`;
